Packs/Code42/Integrations/Code42/Code42_test.py

```diff
@@ -1339,50 +1355,96 @@ def test_highriskemployee_get_all_command_when_no_employees(code42_high_risk_emp
-    _, _, res = highriskemployee_remove_risk_tags_command(
-        client, {"username": "user1@example.com", "risktags": ["FLIGHT_RISK", "CONTRACT_EMPLOYEE"]}
+    _, outputs, res = highriskemployee_remove_risk_tags_command(
+        client, {"username": _TEST_USERNAME, "risktags": ["FLIGHT_RISK", "CONTRACT_EMPLOYEE"]}
     )
-    expected_user_id = "123412341234123412"  # value found in GET_USER_RESPONSE
-    assert res == expected_user_id
+    assert res == _TEST_USER_ID
+    assert outputs["Code42.HighRiskEmployee"]["UserID"] == _TEST_USER_ID
+    assert outputs["Code42.HighRiskEmployee"]["Username"] == _TEST_USERNAME
+    assert outputs["Code42.HighRiskEmployee"]["RiskTags"] == tags
     code42_sdk_mock.detectionlists.remove_user_risk_tags.assert_called_once_with(
-        expected_user_id, ["FLIGHT_RISK", "CONTRACT_EMPLOYEE"]
+        _TEST_USER_ID, ["FLIGHT_RISK", "CONTRACT_EMPLOYEE"]
     )
 
 
 def test_security_data_search_command(code42_file_events_mock):
     client = create_client(code42_file_events_mock)
-    _, _, res = securitydata_search_command(client, MOCK_SECURITY_DATA_SEARCH_QUERY)
-    assert len(res) == 3
+    _, outputs, res = securitydata_search_command(client, MOCK_SECURITY_DATA_SEARCH_QUERY)
+    outputs_list = outputs["Code42.SecurityData(val.EventID && val.EventID == obj.EventID)"]
     actual_query = code42_file_events_mock.securitydata.search_file_events.call_args[0][0]
     filter_groups = json.loads(str(actual_query))["groups"]
-    assert filter_groups[0]["filters"][0]["term"] == "md5Checksum"
-    assert filter_groups[0]["filters"][0]["value"] == "d41d8cd98f00b204e9800998ecf8427e"
-    assert filter_groups[1]["filters"][0]["term"] == "osHostName"
-    assert filter_groups[1]["filters"][0]["value"] == "DESKTOP-0001"
-    assert filter_groups[2]["filters"][0]["term"] == "deviceUserName"
-    assert filter_groups[2]["filters"][0]["value"] == "user3@example.com"
-    assert filter_groups[3]["filters"][0]["term"] == "exposure"
-    assert filter_groups[3]["filters"][0]["value"] == "ApplicationRead"
+    expected_query_items = [
+        ("md5Checksum", "d41d8cd98f00b204e9800998ecf8427e"),
+        ("osHostName", "DESKTOP-0001"),
+        ("deviceUserName", "user3@example.com"),
+        ("exposure", "ApplicationRead")
+    ]
+    expected_file_events = json.loads(MOCK_SECURITY_EVENT_RESPONSE)["fileEvents"]
+
+    # Assert that the  correct query gets made
+    assert len(filter_groups) == len(expected_query_items)
+    for i in range(0, len(filter_groups)):
+        _filter = filter_groups[i]["filters"][0]
+        assert _filter["term"] == expected_query_items[i][0]
+        assert _filter["value"] == expected_query_items[i][1]
+
+    assert len(res) == len(outputs_list) == 3
+    assert res == expected_file_events
+
+    # Assert that the Outputs are mapped from the file events.
+    for i in range(0, len(expected_file_events)):
+        mapped_event = map_to_code42_event_context(expected_file_events[i])
```

Comment:
I know that this is a violation of good unit testing practices since it is calling this mapping method in the expected result --- but file events are very large.....

Reply:
is this method also tested independently? If that method doesn't work but this method passes, that could lead to a hard-to-diagnose problem.

Reply:
but if we have a different test that proves that it works, its probably okay to use here

Reply:
Yes that method is tested